### ISSUE TITLE
Line numbers are getting munged

### ISSUE THREAD

**Issue by Qix-** [Qix-/better-exceptions#11]
Not entirely sure how to reproduce this minimally, but with better_exceptions enabled:

```
Traceback (most recent call last):
  File "test_color.py", line 1, in <module>
    from better_exceptions import color
  File "/src/qix-.better-exceptions/better_exceptions/color.py", line 64, in <module>
    SUPPORTS_COLOR = True
    └ None
TypeError: hex() argument can't be converted to hex
```

and without them enabled:

```
Traceback (most recent call last):
  File "test_color.py", line 1, in <module>
    from better_exceptions import color
  File "/src/qix-.better-exceptions/better_exceptions/color.py", line 51, in <module>
    print hex(magic_number)
TypeError: hex() argument can't be converted to hex
```

**Comment by paradoxxxzero** [comment]
Strange, the traceback module also seems to get lineno this way: https://github.com/python/cpython/blob/master/Lib/traceback.py#L306

**Comment by paradoxxxzero** [comment]
In fact you get the frame f_lineno, what you surely want is the traceback tb_lineno.

**Comment by Qix-** [comment]
You'd think they'd be the same thing... I'll look into this. Thanks for the information @paradoxxxzero, as always!

**Comment by Qix-** [comment]
Reproducible test case in 24b18886fc40fd507965ba62b4b5a24cf57d1ca3 by running `python test_bug.py`.

**Comment by Qix-** [comment]
Fix released as `0.1.5`!
